docs/topics/mta-cli/ref_mta-cli-analyze-flags.adoc

```diff
@@ -13,7 +13,12 @@ The following are the options that you can use together with the `mta-cli analyz
 |====
 |Option|Description
 |`--analyze-known-libraries` (bool)|Analyze open-source libraries.
-|`--context-lines` (int)|A number of lines of source code to include in the output for each incident. The default is 100.
+|`--disable-maven-search`| Set the flag to `true` to disable {ProductShortName} from relying on the Maven search index to determine if a dependency is publicly available (such as an open-source dependency) or internal to the Java binary application during analysis. 
+
+When you disable Maven search, {ProductShortName} at first tries to determine dependencies from the the JAR file's POM file (if any). If this method does not succeed, {ProductShortName} goes through the directory structure to determine dependencies. This method may not produce a reliable dependency classification since the package structure can differ from what is expected by {ProductShortName}. You may see more dependencies in the `dependencies.yaml` file because some dependencies may be wrongly classified as internal.
```

Comment:
Instead of noting dependencies in `dependencies.yaml`, it would probably be better to mention this could cause more incidents vs using maven search.